npm/private/npm_translate_lock_helpers.bzl

```diff
@@ -197,6 +197,44 @@ def _get_npm_auth(npmrc, npmrc_path, environ):
 
     return (registries, auth)
 
+################################################################################
+def _select_npm_auth(url, npm_auth):
+    registry = url.split("//", 1)[-1]
+
+    # Get rid of the port number
+    registry_no_port = registry
+    if ":" in registry:
+        components = registry.split(":", 1)
+        base = components[0]
+
+        path = components[1].split("/", 1)
+        if len(path) == 2:
+            registry_no_port = base + "/" + path[1]
```

Comment:
any reason for the additional `registry_no_port` here? can just assign back to `registery` instead?

Reply:
I see. Both `registery` and `registry_no_port` are needed as they are both checked against on line 228 👍